godot-macros/src/docs.rs

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright (c) godot-rust; Bromeon and contributors.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+mod markdown_converter;
+
+use crate::class::{ConstDefinition, Field, FuncDefinition, SignalDefinition};
+use proc_macro2::{Ident, TokenStream};
+use quote::ToTokens;
+use venial::*;
+
+pub fn make_definition_docs(
+    base: String,
+    description: Vec<Attribute>,
+    members: Vec<Field>,
+) -> TokenStream {
+    let desc = make_docs_from_attributes(description);
+    let members = members
+        .into_iter()
+        .filter(|x| x.var.is_some() | x.export.is_some())
+        .map(member)
+        .collect::<String>();
+    quote::quote! { ::godot::docs::DefinitionDocs { base: #base, description: #desc, members: #members } }
+}
+
+pub fn make_inherent_impl_docs(
+    functions: Vec<FuncDefinition>,
+    constants: Vec<ConstDefinition>,
+    signals: Vec<SignalDefinition>,
+) -> TokenStream {
+    let methods = functions
+        .into_iter()
+        .map(make_method_docs)
+        .collect::<String>();
+    let signals = signals
+        .into_iter()
+        .map(make_signal_docs)
+        .collect::<String>();
+    let constants = constants
+        .into_iter()
+        .map(|ConstDefinition { raw_constant: x }| x)
+        .map(constant)
+        .collect::<String>();
+    quote::quote! { ::godot::docs::ImplementationDocs { methods: #methods, signals: #signals, constants: #constants } }
```

Comment:
```suggestion
    quote! { 
        ::godot::docs::ImplementationDocs { methods: #methods, signals: #signals, constants: #constants } 
    }
```
Just so `quote!` look the same. The field decls could also be broken into multiple lines, not very important though.